src/plugins/intel_gpu/src/graph/include/program_node.h

```diff
@@ -351,10 +352,18 @@ struct program_node {
     bool need_lockable_memory() const;
 
     std::string get_unique_id() const { return unique_id; }
-    void set_unique_id(std::string id) { unique_id = id; }
+    void set_unique_id(std::string id) {
+        unique_id = id;
+        if (myprog.get_parent_primitive() != "") {
+            auto loop_prim = myprog.get_parent_primitive();
+            std::replace(loop_prim.begin(), loop_prim.end(), '/', '_');
+            std::replace(loop_prim.begin(), loop_prim.end(), ':', '_');
```

Comment:
I think those two are not the only prohibited symbols for kernel name... Is it possible to use processing ID of parent primitive instead?

Reply:
There is no processing id -like thing currently, but I was thinking of changing this to be "body_network_id" like thing, how do you think? 
I.e., 
- Let the program have body_prog_id  (e.g., if the program has 3 loops, ids 0-2 exists for that program) 
- when build_body_network, set the body_prog_id
- Use it in its unique_id

Reply:
I just decided to use unique_id as a temporally generated number same as the original behavior to make it simple. I tried the above mentioned impl but thought again, there are too many "_id"s .. maybe it will be better to let the program have parent / body impl more structured way. However at this moment it is only needed for distinguish the entry point, no other usage for them. Please take a look..

Reply:
So I am gonna revert the changes w.r.t the parent_id things, too, soon

Reply:
BTW I now think to set unique_id at the creation time of program_node, instead of at the choose_impl. I'll update soon.